sdk/tests/conformance2/samplers/samplers.html

```diff
@@ -174,7 +174,7 @@
         var pname = testCases[ii].pname;
         var param = testCases[ii].param;
         wtu.shouldGenerateGLError(gl, gl.NO_ERROR, "gl.samplerParameteri(s, " + pname + ", " + param + ")");
-        shouldBe("gl.getSamplerParameter(s, " + pname + ")", "gl['" + enumToString(param) + "']");
+        shouldBe("gl.getSamplerParameter(s, " + pname + ")", "gl['" + (enumToString(param).split('/'))[0] + "']");
```

Comment:
This is a weird use of this function. Let's just do:
`shouldBe("gl.getSamplerParameter(s, " + pname + ")", param);`

Reply:
I changed it. Though https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/js/js-test-pre.js#L372 
WARN: shouldBe() expects string arguments?